### ISSUE TITLE
Is the service down?

### ISSUE THREAD

**Issue by whyvra** [hardillb/node-red-contrib-googlehome-web#88]
Hi,

It seems that I can login via the web portal but Google Assistant isn't able to control the smart devices. It looks like it works at first, but then I get a "Oops, something when wrong".

Thanks

**Comment by hardillb** [comment]
Nope, the service is fine, not running under much load at the moment.

In Node-RED are the nodes showing as connected?
